CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -219,6 +244,19 @@ local function AddTool(tool)
 	slot.text.ClipsDescendants = true
 	slot.text.Selectable = false
 
+	local selectionObject = Util:Create'ImageLabel'
+	{
+		Name = 'SelectionObject';
+		Size = UDim2.new(1,0,1,0);
+		BackgroundTransparency = 1;
+		Image = "rbxasset://textures/ui/Keyboard/key_selection_9slice.png";
```

Comment:
Good to know. He's also working on a new design for it, if I'm not mistaken. I'll update it when he's done with that; I also made the buttons expand themselves when their tool is being used for now.